packages/rum-core/src/boot/startRum.ts

```diff
@@ -223,6 +223,7 @@ export function startRum(
     stop: () => {
       cleanupTasks.forEach((task) => task())
     },
+    hooks,
```

Comment:
Discussed with @amortemousque, this PR expose the `hooks` object in the results of `startRum`.